cmd/posix.go

```diff
@@ -1208,76 +1205,28 @@ func (s *posix) CreateFile(volume, path string, fileSize int64, r io.Reader) (er
 		return err
 	}
 
+	defer w.Close()
```

Comment:
Is there any reason we don't need `w.Sync()` in this implementation?

Reply:
Sync() is relaxed for performance requirements.